stormworksguide.md

```diff
@@ -0,0 +1,48 @@
+# Stormworks Setup Guide
+
+## Linux Support
+
+The server requires Wine to run on Linux, Preferably Wine 8. Using a container avoids the need to install this dependency on the host, otherwise,  Wine is required to be installed on the host system to run the server. 
+
+## Player Permissions
+
+There are 4 settings for player permissions. Admins, Authorized, Blacklist, and Whitelist. 
+
+### Admins
+
+Admins are able to run the in game commands that are listed below, as well as change the settings in the Settings Menu, if enabled in the Stormworks - World settings. 
+
+### Authorized
+
+ In game, players have to be authorized in order to use the workbench, spawn in creatations, ect. 
+
+### Blacklist
+
+Simply a ban list, players can be banned in game, however, you have to remove their id from the ban list, similar to adding a player to the whitelist, however remove the `<id value="TheirID" />`. 
```

Comment:
`Simply a ban list. Players`

Can you provide details of which file has these values in it that will need removed?